components/compass-runtime-agent/README.md

```diff
@@ -0,0 +1,37 @@
+# Compass Runtime Agent
+
+## Overview
+
+This is the repository for the Kyma Compass Runtime Agent.
+
+The main responsibilities of the Compass Runtime Agent are:
+- Establishing a trusted connection between the Runtime and the Compass
+- Renewing the trusted connection between the Runtime and the Compass
+- Configuring the Runtime
+
+
+## Prerequisites
+
+The Compass Runtime Agent requires Go 1.8 or higher.
+
+## Usage
+
+This section explains how to use the Compass Runtime Agent.
+
+### Start the Compass Runtime Agent
```

Comment:
I suggest to add a blank line below the title. Some markdown linters could get angry. Same thing after the 36th line, just before the bullet point